src/main/java/se/liu/ida/hefquin/engine/wrappers/graphqlwrapper/data/GraphQLField.java

```diff
@@ -14,6 +14,7 @@ public interface GraphQLField {
 
     /**
      * @return the GraphQL value type of the field (String,Int,... etc.)
+     * Note: does not include List or Non-nullable identifiers
```

Comment:
this means that the method returns null in such cases? or the empty string? or an error?

Reply:
What I intended for the comment to mean was if a GraphQL type has a field `books: [Book!]!` for example, then the function `getValueType()` would return `Book` and not `[Book!]!`.